lib/train/transports/winrm.rb

```diff
@@ -45,6 +45,9 @@ class WinRM < Train.plugin(1)
     option :port
     option :user, default: 'administrator', required: true
     option :password, nil
+    option :transport, default: :negotiate
```

Comment:
For now, let's add a --winrm- prefix here, to match what InSpec will be sending.  I think in the future we may be able to something like having InSpec strip the prefix from options (or all options would have the prefix - point being it should be consistent)

Reply:
resolved.